linux_sql/README.md

```diff
@@ -1,4 +1,246 @@
-#Linux Cluster Monitoring Agent
-This project is under development. Since this project follows
-the GitFlow, the final work will be merged to the master branch
-after Team Code Team
+# Linux Clustering Monitoring Agent
+## 1.0 Introduction
```

Comment:
good job..

Suggestion: you don't need to number title as it's hard to manage when you add new titles.